zokrates_book/src/concepts/imports.md

```diff
@@ -38,10 +38,10 @@ from "./path/to/my/module" import main as module
 Note that this legacy method is likely to be become deprecated, so it is recommended to use the preferred way instead.
 ### Symbols
 
-Two type of symbols can be imported
+Two type of symbols can be imported:
```

Comment:
```suggestion
Two types of symbol can be imported:
```

is this correct english

Reply:
According to https://english.stackexchange.com/questions/5539/types-of-things-vs-types-of-thing
I think it should be `types of symbols`

Reply:
fair enough
```suggestion
Two types of symbols can be imported:
```

Reply:
That's how it is right now